Gemfile

```diff
@@ -4,7 +4,7 @@ ruby RUBY_VERSION
 
 DECIDIM_VERSION = "~> 0.16.0"
 
-gem 'rails', '5.2.1'
+gem 'rails', '5.2.2.1'
```

Comment:
Add a comment explaining why we add this and when it can be removed please.

Reply:
Done